extensions/amp-story/amp-story.md

```diff
@@ -110,6 +110,66 @@ The new attributes:
 | `poster-square-src` | The story poster in square format (1x1 aspect ratio). |
 | `poster-landscape-src` | The story poster in landscape format (4x3 aspect ratio). |
 
+#### `publisher-logo-src` guidelines
+
+The following guidelines apply to the image for the publisher logo:
+
+- The file must be a raster file, such as `.jpg`, `.png`, or `.gif`.  Don't use vector files, such as `.svg` or `.eps`.
+- Don't use animation.
+- The graphic part of the logo should be legible on the background color.
+
+<table>
+  <tr>
+    <td>
+      <amp-img alt="Logo with blue text on white background"
```

Comment:
I would recommend that you resize your logo files so they are the same dimension, and then amend the syntax here to reflect those dimensions.  Afterall, you are providing guidelines. :)

Reply:
Done